libs/parallelism/futures/include/hpx/futures/packaged_continuation.hpp

```diff
@@ -35,6 +35,7 @@ namespace hpx { namespace lcos { namespace detail {
     HPX_FORCEINLINE void transfer_result(
         SourceState&& src, DestinationState const& dest)
     {
+        hpx::intrusive_ptr<DestinationState> keep_alive(&dest);
```

Comment:
Ok, I'm definitely not making things better. This doesn't compile because `DestinationState` is already an intrusive pointer. Do we really need another one? Is dest always an intrusive pointer?

Reply:
Yeah, you're right. We don't need the keep-alive here.

Reply:
I re-merged, leaving this one out. Could you please re-check?

Reply:
LGTM, thanks Mikael!